jpo-ode-plugins/src/main/java/us/dot/its/jpo/ode/plugin/j2735/OdeTravelerInformationMessage.java

```diff
@@ -30,998 +32,1090 @@
+      public void setRegulatorID(int regulatorID) {
+        this.regulatorID = regulatorID;
       }
 
-      public void setLaneAngle(BigDecimal laneAngle) {
-         this.laneAngle = laneAngle;
+      public String getName() {
+        return name;
       }
 
-      public SpeedLimits[] getSpeedLimits() {
-         return speedLimits;
+      public void setName(String name) {
+        this.name = name;
       }
 
-      public void setSpeedLimits(SpeedLimits[] speedLimits) {
-         this.speedLimits = speedLimits;
-      }
-   }
+    }
 
-   public static class Attributes extends OdeObject {
+    /**
+     * RoadSignID.
+     */
+    @EqualsAndHashCode(callSuper = false)
+    public static class RoadSignID extends OdeObject {
 
-      private static final long serialVersionUID = -6476758554962944513L;
-      private LocalNode[] localNodes;
-      private DisabledList[] disabledLists;
-      private EnabledList[] enabledLists;
-      private DataList[] dataLists;
-      private BigDecimal dWidth;
-      private BigDecimal dElevation;
+      private static final long serialVersionUID = 1L;
 
-      public LocalNode[] getLocalNodes() {
-         return localNodes;
-      }
+      private OdePosition3D position;
+      private String viewAngle;
+      private MutcdCode.MutcdCodeEnum mutcdCode;
+      private String crc;
 
-      public void setLocalNodes(LocalNode[] localNodes) {
-         this.localNodes = localNodes;
+      public OdePosition3D getPosition() {
+        return position;
       }
 
-      public DisabledList[] getDisabledLists() {
-         return disabledLists;
+      public void setPosition(OdePosition3D position) {
+        this.position = position;
       }
 
-      public void setDisabledLists(DisabledList[] disabledLists) {
-         this.disabledLists = disabledLists;
+      public String getViewAngle() {
+        return viewAngle;
       }
 
-      public EnabledList[] getEnabledLists() {
-         return enabledLists;
+      public void setViewAngle(String viewAngle) {
+        this.viewAngle = viewAngle;
       }
 
-      public void setEnabledLists(EnabledList[] enabledLists) {
-         this.enabledLists = enabledLists;
+      public MutcdCode.MutcdCodeEnum getMutcdCode() {
+        return mutcdCode;
       }
 
-      public DataList[] getDataLists() {
-         return dataLists;
+      public void setMutcdCode(MutcdCode.MutcdCodeEnum mutcdCode) {
+        this.mutcdCode = mutcdCode;
       }
 
-      public void setDataLists(DataList[] dataLists) {
-         this.dataLists = dataLists;
+      public String getCrc() {
+        return crc;
       }
 
-      public BigDecimal getdWidth() {
-         return dWidth;
+      public void setCrc(String crc) {
+        this.crc = crc;
       }
+    }
 
-      public void setdWidth(BigDecimal dWidth) {
-         this.dWidth = dWidth;
-      }
+    /**
+     * MsgId.
+     */
+    @EqualsAndHashCode(callSuper = false)
+    public static class MsgId extends Asn1Object {

```

Comment:
I'm not sure we need to tackle it in this PR, but we'll probably want to follow convention and pull each of these classes out to their own file

Reply:
That definitely sounds like a good idea. I've opened a [work item](https://dev.azure.com/SOC-OIT/CDOT/_workitems/edit/279962) for this and put it on the backlog.